.github/workflows/helium/lib/createHeliumPkgFile.js

```diff
@@ -0,0 +1,41 @@
+import Promise from "bluebird";
+import stringify from 'json-stringify-pretty-compact';
```

Comment:
It looks like a different style. Could you please run prettier or linter?

Reply:
Okay. I'll run the linter.